src/table/table-body.jsx

```diff
@@ -183,10 +183,11 @@ const TableBody = React.createClass({
 
     return React.Children.map(this.props.children, (child) => {
       if (React.isValidElement(child)) {
+        const {selected} = child.props;
         const props = {
           displayRowCheckbox: this.props.displayRowCheckbox,
           hoverable: this.props.showRowHover,
-          selected: this._isRowSelected(rowNumber),
+          selected: selected !== undefined ? selected : this._isRowSelected(rowNumber),
```

Comment:
Hey! Thanks for the PR :+1: can we adjust this quickly to use short circuit evaluation instead of a ternary? Should be golden after that! Ta